### PR TITLE
Add config option to skip already assigned PR's (Closes #118)

### LINKED ISSUE #118: Feature Request: Config option to ignore already assigned PR's

### PR DESCRIPTION
**Pull request by hansott**
Closes #118

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -149,6 +150,15 @@ async function work(body) {
     return;
   }
 
+  if (
+    repoConfig.skipAlreadyAssignedPR
```
Comment (vjeux) [inline server.js]: please put && at the end of the previous line

File: server.js [hunk 2]
```diff
@@ -123,6 +123,7 @@ async function work(body) {
     requiredOrgs: [],
     findPotentialReviewers: true,
     actions: ['opened'],
+    skipAlreadyAssignedPR: false
```
Comment (vjeux) [inline server.js]: nit: missing trailing comma

File: server.js (hunk 1 above)
Comment (vjeux) [inline server.js]:
```js
if (repoConfig.skipAlreadyAssignedPR &&
    data.pull_request.assignee &&
    data.pull_request.assignee.login) {
```

File: server.js [hunk 3]
```diff
@@ -149,6 +150,15 @@ async function work(body) {
     return;
   }
 
+  if (
+    repoConfig.skipAlreadyAssignedPR
+    && data.pull_request.assignee
+    && data.pull_request.assignee.login
+  ) {
+    console.log('Skipping because Pull request is already assigned.');
```
Comment (vjeux) [inline server.js]: remove capital from Pull, seems overkill since Request isn't capitalized

**Comment by vjeux** [comment]
Thanks, I'll merge as soon as small nits are fixed

File: server.js (hunk 3 above)
Comment (hansott) [inline server.js]: Fixed in 4622b9c48b52ab1e1575a453502ff07c41601771

File: server.js (hunk 2 above)
Comment (hansott) [inline server.js]: Fixed in 6edd4702793feff3c5ac4cd21c87b330a39e79c9

File: server.js (hunk 1 above)
Comment (hansott) [inline server.js]: Fixed in 691016486e8b53a7d46b48c4a43556890c332d25

**Comment by vjeux** [comment]
Thanks :)

**Comment by hansott** [comment]
🙇

**Pull request merged**